ui/client/components/graph/node-modal/editors/expression/StringEditor.js

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+import LabeledInput from "../field/LabeledInput"
+import PropTypes from "prop-types"
+
+export default function StringEditor(props) {
+
+  const {fieldLabel, renderFieldLabel, expressionObj, onValueChange} = props
+
+  const defaultQuotationMark = "'"
+  const expressionQuotationMark = expressionObj.expression.charAt(0)
+  const quotationMark = !_.isEmpty(expressionQuotationMark) ? expressionQuotationMark : defaultQuotationMark
+
+  const format = (value) => quotationMark + value + quotationMark
+
+  const trim = (value) => value.substring(1, value.length - 1)
+
+  return <LabeledInput
+    {...props}
+    renderFieldLabel={() => props.renderFieldLabel(props.fieldLabel)}
+    onChange={(event) => onValueChange(format(event.target.value))}
+    value={trim(expressionObj.expression)}
+  />
+
+}
+
+export const parseableString = (expressionObj) => {
+  const expression = expressionObj.expression
+  const language = expressionObj.language
+  return stringPattern.test(expression) && language === "spel"
+}
+
+const stringPattern = /(^['].*'$)|(^".*"$)/
```

Comment:
the second part should also contain ^["]? Let's also add (at least) comment that we don't handle expressions with escaped '/"...

Reply:
[] brackets are actually unnecessary, I've added also comment about ecaped characters